src/main/java/uk/gov/justice/probation/courtcasematcher/model/domain/Sex.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.probation.courtcasematcher.model.domain;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Optional;
+
+@Slf4j
+public enum Sex {
+    FEMALE("F"),
+    MALE("M"),
+    NOT_KNOWN("N"),
+    NOT_SPECIFIED("NS");
+
+    private final String name;
+
+    Sex(final String sex) {
+        this.name = sex;
+    }
+
+    public static Sex getNormalisedSex(final String sex) {
+
+        switch (Optional.ofNullable(sex).map(s -> s.trim().toUpperCase()).orElse(NOT_KNOWN.name())) {
+            case "MALE", "M" -> { return Sex.MALE; }
+            case "FEMALE", "F" -> { return Sex.FEMALE; }
+            case "NOT_KNOWN", "N" -> { return Sex.NOT_KNOWN; }
+            case "NOT_SPECIFIED", "NS" -> { return Sex.NOT_SPECIFIED; }
+            default -> {
```

Comment:
default is never reachable as the switch expression already 

`        return switch (Optional.ofNullable(sex).map(s -> s.trim().toUpperCase()).orElse(NOT_KNOWN.name())) {
            case "MALE", "M" -> Sex.MALE;
            case "FEMALE", "F" -> Sex.FEMALE;
            case "NOT_KNOWN", "N" -> Sex.NOT_KNOWN;
            case "NOT_SPECIFIED", "NS" -> Sex.NOT_SPECIFIED;
        };`

Reply:
Yeh, Copied from CCS, let me have a look now.

Reply:
Nope, it didn't work. Sorry

Reply:
`error: the switch expression does not cover all possible input values`